accelerate/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = [ "setuptools >= 61.0", "Cython >= 0.28", "numpy" ]
```

Comment:
Cython 3.0 (still beta) has some significant changes -- we should probably pin to less than 3.0 here.

Reply:
Cython v3 is actually in alpha at this time, but I've yet to see any issues with v3 with the projects I use Cython for. When Cython v3 does release, I expect `pyopengl` can set a pin or fix compatibility if issues occur

Reply:
Update: Cython v3 released in July 2023

Reply:
Maybe update to Cython == 3.0 then?

Unless we're testing with both, it's probably better to pin it.

Reply:
Perhaps in another merge-request